relayer/spy_relayer/src/backends/swim/listener.ts

```diff
@@ -134,6 +134,14 @@ export class SwimListener implements Listener {
     return currentNumRequests >= 0 ? currentNumRequests < env.requestLimit : false;
   }
 
+  /**
+   * Sanity check the maxSwimUSDFee field.
+   * To prevent exploits, we automatically reject any payload with a maxSwimUSDFee that is less than 0.01 swimUSD (10000)
+   */
+  verifyMaxSwimUSDFeeIsValid(payload: ParsedTransferWithArbDataPayload<ParsedSwimData>): boolean {
```

Comment:
1/ Should we log anything if we reject an invalid maxSwimUSDFee?
2/ Can you add a TODO here that vaguely denotes all the conditions we wanna check (doesn't have to be super specific or anything). In an ideal world, we'll want to test this method for all conditions.

Reply:
1. I can add a debug log
2. Yeah i'll add a TODO